multiqc/modules/goleft_indexcov/goleft_indexcov.py

```diff
@@ -0,0 +1,144 @@
+        except ValueError:
+            if chrom in set(["X"]):
+                return chrom
+
+    def coverage_plot(self):
+        max_y = 2.5
+        data = collections.defaultdict(lambda: collections.defaultdict(dict))
+        for fn in self.find_log_files(config.sp["goleft_indexcov"]["coverage"], filehandles=True):
+            header = fn['f'].readline()
+            sample_names = [self._clean_name(x) for x in header.strip().split()[3:]]
+            for parts in (l.rstrip().split() for l in fn['f']):
+                chrom, start, end = parts[:3]
+                sample_covs = parts[3:]
+                if self._short_chrom(chrom) is not None:
+                    for cov, sample in zip(sample_covs, sample_names):
+                        data[chrom][sample][int(end) + int(start) // 2] = min(max_y, float(cov))
+
+        if data:
+            chroms = sorted(data.keys(), key=self._short_chrom)
+            pconfig = {
+                'id': 'goleft_indexcov-coverage-plot',
+                'title': 'Scaled coverage by chromosome',
+                'ylab': 'Scaled coverage',
+                'xlab': 'Position',
+                'ymin': 0,
+                'ymax': max_y,
+                'xDecimals': False,
+                'data_labels': [{"name": self._short_chrom(c)} for c in chroms]}
+            return linegraph.plot([data[c] for c in chroms], pconfig)
+
+    def roc_plot(self):
+        desc = '<p>Coverage (ROC) plot that shows genome coverage at at given (scaled) depth. \n\
+        Lower coverage samples have shorter curves where the proportion of regions covered \n\
+        drops off more quickly. This indicates a higher fraction of low coverage regions. \n\
+        </p>'
+        data = collections.defaultdict(lambda: collections.defaultdict(dict))
+        for fn in self.find_log_files(config.sp["goleft_indexcov"]["roc"], filehandles=True):
+            header = fn['f'].readline()
+            sample_names = [self._clean_name(x) for x in header.strip().split()[2:]]
+            for parts in (l.rstrip().split() for l in fn['f']):
+                chrom, cov = parts[:2]
+                sample_vals = parts[2:]
+                if self._short_chrom(chrom) is not None:
+                    for val, sample in zip(sample_vals, sample_names):
+                        data[chrom][sample][float(cov)] = float(val)
+        if data:
+            chroms = sorted(data.keys(), key=self._short_chrom)
+            pconfig = {
+                'id': 'goleft_indexcov-roc-plot',
+                'title': 'ROC: genome coverage per scaled depth by chromosome',
+                'xlab': 'Scaled coverage',
+                'ylab': 'Proportion of regions covered',
+                'ymin': 0, 'ymax': 1.0,
+                'xmin': 0, 'xmax': 1.5,
+                'data_labels': [{"name": self._short_chrom(c)} for c in chroms]}
+            return desc + linegraph.plot([data[c] for c in chroms], pconfig)
+
+    def _clean_name(self, name):
```

Comment:
Should use [`self.clean_s_name`](https://github.com/ewels/MultiQC/blob/master/multiqc/modules/base_module.py#L108-L149) instead from the main Base module. Does a load of extra things then (user config cleaning, prepending directory names etc).